libraries/AP_Mount/AP_Mount_SToRM32.cpp

```diff
@@ -110,8 +110,8 @@ void AP_Mount_SToRM32::find_gimbal()
         return;
     }
 
-    // return if search time has has passed
-    if (AP_HAL::millis() > AP_MOUNT_STORM32_SEARCH_MS) {
+    // search for gimbal for 60 seconds or until armed
+    if ((AP_HAL::millis() > AP_MOUNT_STORM32_SEARCH_MS) && hal.util->get_soft_armed()) {
```

Comment:
Why not just get_soft_armed?

ie.  just not search for gimbals if we are armed.

Reply:
Hi @peterbarker,

This is consistent with the existing gremsy code so I think we should go with this (or change them both).  But my guess is that this avoids problems with Planes and Rovers that might have ARMING_REQUIRED = 0.  They might become armed before the gimbal starts communicating

Reply:
> Hi @peterbarker,
> 
> This is consistent with the existing gremsy code so I think we should go with this (or change them both). But my guess is that this avoids problems with Planes and Rovers that might have ARMING_REQUIRED = 0. They might become armed before the gimbal starts communicating

Fair point.